src/app/events/org-events.component.ts

```diff
@@ -168,4 +181,8 @@ export class OrgEventsComponent implements OnInit {
   private gotoSearchEvents() {
     this.tabGroup.selectedIndex = 1
   }
+
+  public isPublicView() {
+    return !remult.authenticated() // TODO are volunteers authenticated?
```

Comment:
volunteers (drivers) are authenticated 

Not sure if drivers should also see other drivers trips as if they were not authenticated (looking for a ride for a friend etc...)